### PR TITLE
Guide components dynamic async

### PR DESCRIPTION
**Pull request by FLarra**
Some specific translations I decided to pick:

```
callback > función callback
```

### CODE REVIEW

File: src/v2/guide/components-dynamic-async.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ---
-title: Dynamic & Async Components
+title: Componentes Dinámicos & Asíncronos
 type: guide
 order: 105
 ---
 
-> This page assumes you've already read the [Components Basics](components.html). Read that first if you are new to components.
+> Esta página asume que usted ya ha leído [Conceptos Básicos de Componentes](components.html). Léa eso primero si usted es nuevo con respecto a componentes.
```
Comment (sxcamacho) [inline src/v2/guide/components-dynamic-async.md]: Please change "Léa" to "Lea"

Comment (sxcamacho) [reply src/v2/guide/components-dynamic-async.md]: The rest of the article looks good. Good job!

**Comment by FLarra** [comment]
Thanks @sxcamacho for your feedback!

I got inspired by the text found in `components-custom-events.md` which was written as `Léa`, considering your comment decided to fix that in both: `components-custom-events.md` and `components-dynamic-async.md` and also wrote those texts in the same way since both share the same message.

**Comment by sxcamacho** [comment]
> components-custom-events

Yes, please change for both cases.

**Pull request merged**